raiden/network/proxies/utils.py

```diff
@@ -139,12 +139,14 @@ def raise_on_call_returned_empty(given_block_identifier: BlockSpecification) ->
     # We know that the given address has code because this is checked
     # in the constructor
     if isinstance(given_block_identifier, T_BlockHash):
```

Comment:
You could use `format_block_id` here as well

Reply:
good catch